coap-gateway/service/publishedResources.go

```diff
@@ -14,9 +14,11 @@ type publishedResources struct {
 }
 
 func (p *publishedResources) Add(href ...string) {
-	p.lock.Lock()
-	defer p.lock.Unlock()
-	p.resources = p.resources.Insert(href...)
+	/*
```

Comment:
This probably doesn't belong in this PR.

Reply:
fixed